Makefile.3ds

```diff
@@ -176,9 +176,13 @@ endif
 $(OUTPUT).3dsx: $(OUTPUT).elf
 $(OUTPUT).elf:  $(OFILES)
 
-#$(OUTPUT).smdh	:	$(APP_ICON)
-#	@bannertool makesmdh -s "$(APP_TITLE)" -l "$(APP_DESCRIPTION)"  -p "$(APP_AUTHOR)" -i $(APP_ICON) -o $@
-#	@echo "built ... $(notdir $@)"
+$(OUTPUT).smdh	:	$(APP_ICON)
+	@bannertool makesmdh -s "$(APP_TITLE)" -l "$(APP_DESCRIPTION)"  -p "$(APP_AUTHOR)" -i $(APP_ICON) -o $@
+	@echo "built ... $(notdir $@)"
+
+$(TARGET).bnr	:	$(TOPDIR)/$(BNR_IMAGE) $(TOPDIR)/$(BNR_AUDIO)
+	@bannertool	makebanner -o $@ -i $(TOPDIR)/$(BNR_IMAGE) -a $(TOPDIR)/$(BNR_AUDIO)
+	@echo "built ... $@"
```

Comment:
This is similar to Issue #4. I didn't want a dependency on bannertool since it isn't included with devkitARM. There is already a rule in 3ds_rules for building smdh using smdh_tool, so maybe smdh_tool needs to be fixed since it seems to cause a blank icon :(